google/cloud/aiplatform/datasets/tabular_dataset.py

```diff
@@ -18,7 +18,9 @@
 import csv
 import logging
 
-from typing import List, Optional, Sequence, Tuple, Union
+from typing import List, Optional, Sequence, Set, Tuple, Union
+
+from google.cloud.bigquery.schema import SchemaField
```

Comment:
`SchemaField` seems to be exposed at the package root `bigquery` which is imported below.

Reply:
Removed